packages/pigeon/platform_tests/flutter_null_safe_unit_tests/test/null_safe_test.mocks.dart

```diff
@@ -0,0 +1,55 @@
+// Mocks generated by Mockito 5.0.4 from annotations
+// in flutter_unit_tests/test/null_safe_test.dart.
+// Do not manually edit this file.
+
+import 'dart:async' as _i3;
```

Comment:
oh that's how mockito works these days. huh...

Reply:
Yea, it's kind of messy.  They had to do it in response to null-safety apparently.  It's an interesting look at how to do codegen which is different than the approach we took for pigeon.  With them you have to use `build_runner`.  I ended up just checking in the code anyways because running `build_runner` adds like an extra 30 seconds to the CI script.